apps/address-service/domains/common/utils/services/suggest/SuggestionProviderDetector.js

```diff
@@ -0,0 +1,29 @@
+const {
+    DadataSuggestionProvider,
+    GoogleSuggestionProvider,
+} = require('@address-service/domains/common/utils/services/suggest/providers')
+
+class SuggestionProviderDetector {
+
+    /**
+     * @param {string} geo
+     * @returns {AbstractSuggestionProvider}
+     */
+    getProvider (geo) {
+        /** @type {AbstractSuggestionProvider} */
+        let suggestionProvider
+        switch (geo) {
+            case 'dadata':
```

Comment:
Could you please extract values for all these cases into constants?

Reply:
Done.